### PR TITLE
Add 'psss' hierarchy to report request

### PR DESCRIPTION
**Pull request by rohan-bes**

### CODE REVIEW

File: packages/report-server/.env.example [hunk 1]
```diff
@@ -10,3 +10,4 @@ DHIS_USERNAME=
 JWT_SECRET=
 PORT=
 API_CLIENT_SALT=
+ENTITY_SERVER_API_URL=
```
Comment (kael89) [inline packages/report-server/.env.example:13]: I didn't change the name to `ENTITY_SERVER_API_URL` to be consistent with the existing convention in `psss-server`. Ideally we'd make all such variables consistent across packages!

```suggestion
ENTITY_API_URL=
```

**Review by kael89 — APPROVED** [review]
Looks good (with one small change but pre-approving)!

My only comment is that I can still fetch reports in the current version of `psss-server`, without having to pass a `hierarchy` - no error is thrown.

**Comment by kael89** [comment]
> My only comment is that I can still fetch reports in the current version of `psss-server`, without having to pass a `hierarchy` - no error is thrown.

I actually got the error now after running the latest migrations 🙌

File: packages/report-server/.env.example (hunk 1 above)
Comment (rohan-bes) [reply packages/report-server/.env.example:13]: Yup, I'll change this but it's also `ENTITY_SERVER_API_URL` in `lesmis-server` as well...

Comment (kael89) [reply packages/report-server/.env.example]: True, as discussed in https://github.com/beyondessential/tupaia/pull/2382#discussion_r612213598

For now being consistent across the package should be fine, the next step would be ensuring cross-package consistency

**Pull request merged**